test/erlcloud_ec2_tests.erl

```diff
@@ -18,6 +18,10 @@
 %% The _f macro is a terse way to wrap code in a fun. Similar to _test but doesn't annotate with a line number
 -define(_f(F), fun() -> F end).
 
+-ifdef(TEST).
+-compile(export_all).
+-endif.
```

Comment:
The `ifdef` guard seems unnecessary here, since there's no reason to compile this file when not testing.